client/components/App.jsx

```diff
@@ -36,12 +36,16 @@ class App extends React.Component {
         //get all the rest of the comments once the more button is clicked
         this.setState({ comments });
       });
+
+    let moreBtn = document.querySelector('#more-btn');
```

Comment:
If you ever use the airbnb eslint, you might want to move this into a static function.